pandas/io/formats/csvs.py

```diff
@@ -30,30 +32,54 @@
 
 
 class CSVFormatter:
+    r"""
```

Comment:
I think can delete this docstring; it doesn't add any value yet over what the annotations already tell you